mail_inline_css/views/email_preview.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't get why this change is needed for mail_inline_css :thinking: 
Also, maybe better to open another PR because name, description and discussion are not relevant anymore.

Reply:
I see the updated description now: pls change the name and report the description into a comment in the file.

Reply:
mail_inline_css override generate_email function, if data not there it fails and this is a need for both addons, so or we introduce it on the separate addon, I put it here as this looks more basic than qweb

Reply:
hmm ok, tnx for the explanation. Question: if you have tmpl_qweb installed but not this, would you still have the issue?

Reply:
yes

Reply:
hmm then we I'd propose the fix to odoo core and meanwhile propose the patch for OCB.
As an alternative, could you try this?
1. port https://github.com/OCA/social/pull/534 to v13 (I realized only today that I made it on v11 :stuck_out_tongue: )
2. test if the issue is still there on mail_inline_css (it shouldn't)
3. if point 2 is happy, move the fix to _qweb only